frappe/desk/doctype/bulk_update/bulk_update.js

```diff
@@ -22,8 +22,16 @@ frappe.ui.form.on('Bulk Update', {
 
 					if (failed.length && !r._server_messages) {
 						frappe.throw(__('Cannot update {0}', [failed.map(f => f.bold ? f.bold(): f).join(', ')]));
+					} else {
+						frappe.msgprint({
```

Comment:
Why not add `frm.dashbaord()` to show realtime updation of data ?

Reply:
Progress is shown if docs to be updated are more than 10
![bulk_update_progress](https://user-images.githubusercontent.com/24353136/67750170-b016ec80-fa54-11e9-8691-03f69d4667d5.gif)